src/main/java/org/sagebionetworks/bridge/services/ParticipantFileService.java

```diff
@@ -163,12 +150,14 @@ private String getFilePath(ParticipantFile file) {
     }
 
     private URL generatePresignedRequest(ParticipantFile file, HttpMethod method) {
-        Date expiration = new DateTime().plusMinutes(EXPIRATION_IN_MINUTES).toDate();
+        DateTime expiration = file.getCreatedOn().plusMinutes(EXPIRATION_IN_MINUTES);
```

Comment:
This will break getParticipantFile().

This method is used for both get and create. The intent is that the pre-signed URL should expire 24 hours after the request is generated. However, since get requests can come long after the file is first created, this will generate expiration times in the past.

You want to keep the `new DateTime()`, or better yet, use `DateTime.now()`.